src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 32px 16px 19px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
```

Comment:
It would be great to create variables for all colors per page, so you can change them in one place